sdk-lib/src/main/java/cash/z/ecc/android/sdk/ext/ZcashSdk.kt

```diff
@@ -79,11 +79,21 @@ object ZcashSdk {
      */
     const val SPEND_PARAM_FILE_NAME = "sapling-spend.params"
 
+    /**
+     * Maximum file size for the sapling spend params - 50MB
+     */
+    const val SPEND_PARAM_FILE_MAX_SIZE = 50L * 1024L * 1024L
```

Comment:
We probably don't need these in the public API.  What if we move them to a `companion object` on the SaplingParamTool?

Also maybe consider including units in the field name, e.g. `_BYTES`, to make it more self-documenting.